ui-tests/src/test/resources/features/technical-extension.feature

```diff
@@ -50,8 +50,8 @@ Feature: Upload tech extension and add it to integration
     
   @tech-extension-import-new-tech-extension
   Scenario: Import new technical extensions
-    When Camilla clicks on the "Import technical extension" button
-    Then she is presented with the Syndesis page "Technical Extensions Import"
+    When Camilla clicks on the "Import Technical Extension" button
+    Then she is presented with the Syndesis page "Import Technical Extensions"
```

Comment:
We also need to change name in SyndesisPage.

Reply:
Done.